core/src/main/java/jenkins/util/SystemProperties.java

```diff
@@ -347,7 +374,32 @@ public static Duration getDuration(@NonNull String name, @CheckForNull Duration
         String v = getString(name);
         if (v != null) {
             try {
+                DurationStyle.detectAndParse(v, ChronoUnit.SECONDS);
```

Comment:
Deliberately discarding the result?

Reply:
I thought I had fixed and pushed already :(